proto/executor.proto

```diff
@@ -7,11 +7,17 @@ option java_package = "com.pingcap.tidb.tipb";
 
 import "expression.proto";
 import "schema.proto";
+
 import "gogoproto/gogo.proto";
+import "rustproto.proto";
 
 option (gogoproto.marshaler_all) = true;
 option (gogoproto.sizer_all) = true;
 option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.goproto_unkeyed_all) = false;
+option (gogoproto.goproto_unrecognized_all) = false;
+option (gogoproto.goproto_sizecache_all) = false;
```

Comment:
Do we really need to turn off this option? I think it's helpful for the situation in https://github.com/tikv/client-go/blob/master/internal/locate/region_request.go#L1763-L1775

Reply:
`goproto_sizecache_all` generates `XXX_sizecache`, but it looks like it is never used in TiDB code base.
https://github.com/search?q=org%3Apingcap%20XXX_sizecache&type=code